docbook/reference/en/en-US/modules/Multipart.xml

```diff
@@ -1198,36 +1198,6 @@ String name = proxy.putMultipartForm(new JsonFormResource
             make sure that nothing parses the stream before RESTEasy gets it.
         </para>
     </section>
-    <section id="multipart_overwrite_default_content_type">
-        <title>Overwriting the default fallback content type for multipart messages</title>
-        <para>By default if no Content-Type header is present in a part,
-           <code>"text/plain; charset=us-ascii"</code> is used as the fallback.
-           This is the value defined by the MIME RFC. However some web clients,
-           like most, if not all, web browsers,
-            do not send Content-Type headers for all fields in a multipart/form-data
-           request.  They send them only for the file parts.  This can cause
-           character encoding and unmarshalling errors on the server side. To correct
-           this there is an option to define an other, non-rfc compliant fallback value.
-           This can be done dynamically per request with the PreProcessInterceptor
-            infrastructure of RESTEasy. In the following example we will set
-           <code>"*/*; charset=UTF-8"</code> as the new default fallback:
-            <programlisting>
-<![CDATA[
-import org.jboss.resteasy.plugins.providers.multipart.InputPart;
-
-@Provider
-@ServerInterceptor
-public class ContentTypeSetterPreProcessorInterceptor
-    implements PreProcessInterceptor {
-
-    public ServerResponse preProcess(HttpRequest request, ResourceMethod method)
-            throws Failure, WebApplicationException {
-        request.setAttribute(InputPart.DEFAULT_CONTENT_TYPE_PROPERTY, "*/*; charset=UTF-8");
-        return null;
-    }
-}]]></programlisting>
-        </para>
-    </section>
```

Comment:
While I see the example making use of PreProcessInterceptors, I'm wondering if the multipart related functionality here should still be documented (perhaps the standard JAX-RS 2.0+ interceptors can be used instead?)

Reply:
The code you have flagged is being removed from the chapter.  I'm confused by your comment.

Reply:
@rsearls, I think what I meant when I wrote the comment above was that instead of simply removing the documentation section above because the example describing it make use of classes that have been removed in master, we should revisit the example / description of the feature to use the standard JAX-RS interceptors / filters instead. This assumes that the feature here, as well as those in the comments below, is still available on master. If that is not the case, then yes, erasing the section as you're proposing is fine. Do you see what I mean?